### PR TITLE
✨ bsx series-insight

### LINKED ISSUE #3654: Series insights for Basilisk
We would now need a series of insights for Basilisk.
For now, we just start light, so we can test it out there.
Link it to Stats as Dropdown

| Colection name | Volume | Floor Price | Buys | Offers | Owners | Assets | 
| -- | -- | -- | -- | -- | -- | -- | 
| name of collection | total volume of buys | lowest listed price | amount of buy events | total count of active offers | amount of unique addresses own NFT in collection | number of items in collection | 

Most probably would be needed to make a resolver to https://github.com/kodadot/snek
Let's inspire by https://kodadot.xyz/series-insight we already have majority of queries out there

### PR DESCRIPTION
**Pull request by roiLeo**
- <s>`/bsx/series-insight`</s>

// TODO:
- [x] multi chain `/series-insight`
- [x] toggle logo from chain
- [x] series in bsx navbar
- [x] link bsx topCollection to bsx series
- [ ] chain selection dropdown 

## PR Type

- [x] Feature

## Context

- [x] Closes #3654 

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=GemsyWUUtZU5HAiEQJqqgVz4YkekKXnmutYryBhbcVD4YW3)

## Screenshot 📸

Screenshot 2022-11-25 at 10-26-30 NFT artist rank KodaDot - Kusama NFT Market Explorer

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -825,7 +825,7 @@
   },
   "series": {
     "title": "Top Collection",
-    "subtitle": "The top NFTs on RMRK, ranked by volume, floor price and other statistics.",
+    "subtitle": "The top NFTs on {chain}, ranked by volume, floor price and other statistics.",
```
Comment (Jarsen136) [inline locales/en.json:828]: It seems does not work according to my screenshot.

File: pages/series-insight.vue [hunk 2]
```diff
@@ -3,25 +3,33 @@
...
+        </p>
       </div>
       <div class="column">
         <img
+          v-if="urlPrefix === 'rmrk'"
           src="~/assets/rmrk-logo-pink-faded.png"
           alt="RMRK"
           class="rmrk-logo is-hidden-mobile" />
+        <img
+          v-else
```
Comment (Jarsen136) [inline pages/series-insight.vue:17]: v-else-if `bsx`  would be better

**Comment by roiLeo** [comment]
> As I see the total count of the series collection is 14. Could we remove the range selector or change the number

I can't know the maximum number of collections on this chain (14 right now), I don't see a problem with this component no need to make a condition just for this case.

**Review by Jarsen136 — APPROVED** [review]
LGTM

**Comment by yangwao** [comment]
pay 200 usd 

let's merge it

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $200 @ 30 USD/KSM ~ 6.667 $KSM
🧗 [GemsyWUUtZU5HAiEQJqqgVz4YkekKXnmutYryBhbcVD4YW3](https://kusama.subscan.io/account/GemsyWUUtZU5HAiEQJqqgVz4YkekKXnmutYryBhbcVD4YW3)
🔗 [0x2e858a64ac478ab7731e7958910725a8e0404070a7101facb15c78474a096579](https://kusama.subscan.io/extrinsic/0x2e858a64ac478ab7731e7958910725a8e0404070a7101facb15c78474a096579)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**